### ISSUE TITLE
Weapons models disappearing on other player deaths

### ISSUE THREAD

**Issue by Solunae** [adamdburton/pointshop#67]
Hi,

We are currently running a TTT server with pointshop.

The problem we are seeing is the viewmodel on the guns you carry dissappear. It will still work normally but it won't show the weapon in your hands or any other weapon you equip after that. No errors appear in console. Players around you can still see your weapon.

This problem does not occur when we disable the pointshop by taking out the autorun Lua. (Also disabling pointshop causes ragdolls not to appear when a player is killed.)

As far as we can tell it only happens when another player dies but it doesn't happen all the time. The player that can't see his weapons is random (as far we know).

So basically;
- Random player weapon viewmodel disappears on another players death at random times.

Currently we tell people to start recording a demo and stopping it right after. This will reset the viewmodel and make it visible again.

But I'm looking for a more permanent fix. Any ideas?
